### DISCUSSION TITLE
Hacktoberfest 2023!

### DISCUSSION THREAD

**Discussion by NathanTarbert** [boxyhq/saas-starter-kit#519] [Announcements]
Welcome, let's kick off Hacktoberfest 2023 with a resounding celebration!

`The best way to celebrate and show your support is to star this repo 🙂`

 We've thoughtfully tagged numerous issues with `hacktoberfest`, and there are more exciting tags on the way.

For **Hacktoberfest 2023**, the coveted prize is an exclusive **certificate from BoxyHQ**.

Participating is a breeze – just request to be assigned to one of the tagged issues. All it takes is two successfully merged Pull Requests, and as October draws to a close, your outstanding contributions will be duly rewarded.

**Comment by NathanTarbert** [comment]
Who would like to join in? 
<img src="https://media2.giphy.com/media/3oKHWficpURdVIsxZS/giphy.gif"/>

**Comment by shartrooper** [comment]
That sounds great, I think I have two or more PRs succesfully merged already. 😃
